src/main/java/mekanism/common/CommonPlayerTickHandler.java

```diff
@@ -126,8 +126,9 @@ private void tickEnd(Player player) {
             if (!primaryJetpack.isEmpty()) {
                 JetpackMode primaryMode = ((IJetpackItem) primaryJetpack.getItem()).getJetpackMode(primaryJetpack);
                 JetpackMode mode = IJetpackItem.getPlayerJetpackMode(player, primaryMode, () -> Mekanism.keyMap.has(player.getUUID(), KeySync.ASCEND));
+                double jetpackThrust = ((IJetpackItem) primaryJetpack.getItem()).getJetpackThrust(primaryJetpack);
```

Comment:
This can be moved past the disabled check unless the intention was to also consider it disabled if thrust is zero